src/modules/navigator/mission_block.cpp

```diff
@@ -508,7 +508,12 @@ MissionBlock::mission_item_to_position_setpoint(const mission_item_s &item, posi
 	sp->loiter_radius = (fabsf(item.loiter_radius) > NAV_EPSILON_POSITION) ? fabsf(item.loiter_radius) :
 			    _navigator->get_loiter_radius();
 	sp->loiter_direction = (item.loiter_radius > 0) ? 1 : -1;
-	sp->acceptance_radius = item.acceptance_radius;
+	sp->acceptance_radius = _navigator->get_default_acceptance_radius();
+
+	if (item.acceptance_radius > 0.0f) {
```

Comment:
To be paranoid (until it's documented) could you check ISFINITE as well? Setting the default in else would also be slightly better (only touching sp->acceptance_radius once).